rust/opendp_tooling/src/codegen/python.rs

```diff
@@ -228,9 +230,23 @@ def {then_name}(
         String::new()
     };
 
+    let deprecated = func
+        .deprecated
+        .as_ref()
+        .map(|msg| {
+            if msg.contains("Polars") {
+                format!(
+                    "@deprecated(version='0.11.1', reason='Use `opendp.extras.polars` instead')\n"
```

Comment:
You could change the deprecated attribute in Function to:

```rust
struct Deprecated {
	version: String,
	note: String,
}
```

Then here in this code, you'd craft the message accordingly:

```rust
format!("@deprecated(version={}, reason={})\n" deprecated.version, deprecated.note)
```

This way you don't need to update the code here when you add another deprecation.